yarn.lock

```diff
@@ -12986,7 +12986,7 @@ tsickle@^0.38.0:
   resolved "https://registry.yarnpkg.com/tsickle/-/tsickle-0.38.1.tgz#30762db759d40c435943093b6972c7f2efb384ef"
   integrity sha512-4xZfvC6+etRu6ivKCNqMOd1FqcY/m6JY3Y+yr5+Xw+i751ciwrWINi6x/3l1ekcODH9GZhlf0ny2LpzWxnjWYA==
 
-tslib@2.2.0:
+tslib@2.2.0, tslib@^2.2.0:
```

Comment:
A little weird that these two didn't get collapsed into one.

Reply:
It's because `@angular-devkit/build-optimizer` depends on `tslib@2.2.0` (without the caret). This is correct.